comment/conf/defaults.py

```diff
@@ -26,3 +26,5 @@
 COMMENT_USE_EMAIL_FIRST_PART_AS_USERNAME = False
 
 COMMENT_ALLOW_TRANSLATION = False
+
+COMMENT_ALLOW_SUBSCRIPTION = False
```

Comment:
Shouldn't this feature be enabled by default? I like this option.

Reply:
Hmmmm, No bcz the email configurations is a part of this feature therefore the developer must know what is doing when enable it.

Reply:
Okay, so ideally we should add the link to somewhere in the first page of documentation so that users are aware of this configuration. I think a lot of people might want to incorporate this feature if they can set up this properly.

From my personal experience, mostly I only look extensively at any software's documentation only if I have an issue with it, or I want to configure it a certain way.

Reply:
Let's keep it disabled bu default for now.